tests/test_telemetry.py

```diff
@@ -117,9 +117,6 @@ def test_telemetry_message_required_headers(self):
             check_condition=not_onboarding_event,
         )
 
-    @missing_feature(library="python")
-    @flaky(library="ruby", reason="AIT-8418")
-    @flaky(library="java", reason="AIT-9152")
     def test_seq_id(self):
```

Comment:
Looks like Ruby is still flaky despite this fix
```suggestion
    @flaky(library="ruby", reason="AIT-8418")
    def test_seq_id(self):
```